tests/unit/test_limitless_connection_service.py

```diff
@@ -0,0 +1,550 @@
+#  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#  Licensed under the Apache License, Version 2.0 (the "License").
+#  You may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import psycopg
+import pytest
+
+from aws_advanced_python_wrapper.errors import AwsWrapperError
+from aws_advanced_python_wrapper.host_availability import HostAvailability
+from aws_advanced_python_wrapper.hostinfo import HostInfo, HostRole
+from aws_advanced_python_wrapper.limitless_connection_plugin import (
+    LimitlessConnectionContext, LimitlessConnectionPlugin,
+    LimitlessRouterService)
+from aws_advanced_python_wrapper.utils.messages import Messages
+from aws_advanced_python_wrapper.utils.properties import (Properties,
+                                                          WrapperProperties)
+
+CLUSTER_ID: str = "some_cluster_id"
+EXPIRATION_NANO_SECONDS: int = 60 * 60 * 24 * 7
```

Comment:
This totals to 604800 ns which is .0006 seconds, does it need to be longer? Not sure if its used in any meaningful way but if it is it will expire very quickly